TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -381,14 +381,14 @@ void AddPastItemLocations()
 			Add(new ItemKey(5, 15, 296, 192), "Just an egg", ItemProvider.Get(EItemType.MaxAura), CastleKeep);
 			Add(new ItemKey(5, 41, 72, 160), "Under the twins", ItemProvider.Get(EInventoryEquipmentType.BuckleHat), CastleKeep);
 			Add(new ItemKey(5, 20, 504, 48), "Advisor jump", null, CastleKeep & R.TimeStop);
-			Add(new RoomItemKey(5, 5), "Twins", ItemProvider.Get(EInventoryRelicType.DoubleJump), CastleKeep & R.TimeStop);
+			Add(new RoomItemKey(5, 5), "Golden Idol", ItemProvider.Get(EInventoryRelicType.DoubleJump), CastleKeep & R.TimeStop); //use its actual name for consistency
 			Add(new ItemKey(5, 22, 312, 176), "Royal guard tiny room", ItemProvider.Get(EItemType.MaxSand), CastleKeep & ((R.TimeStop & R.ForwardDash) | R.DoubleJump));
 			areaName = "Royal Towers";
 			Add(new ItemKey(6, 19, 200, 176), "Royal towers floor secret", ItemProvider.Get(EItemType.MaxAura), RoyalTower & R.DoubleJump & OculusRift);
 			Add(new ItemKey(6, 27, 472, 384), "Royal towers pre-climb gap", ItemProvider.Get(EInventoryUseItemType.MagicMarbles), MidRoyalTower);
 			Add(new ItemKey(6, 1, 1512, 288), "Royal towers long balcony", ItemProvider.Get(EInventoryUseItemType.Potion), MidRoyalTower);
-			Add(new ItemKey(6, 25, 360, 176), "Next to bottom royal tower struggle juggle", ItemProvider.Get(EInventoryUseItemType.HiEther), UpperRoyalTower & DoubleJumpOfNpc);
-			Add(new ItemKey(6, 3, 120, 208), "Bottom royal tower struggle juggle", ItemProvider.Get(EInventoryFamiliarType.Demon), UpperRoyalTower & DoubleJumpOfNpc);
+			Add(new ItemKey(6, 25, 360, 176), "Weenie hut junior", ItemProvider.Get(EInventoryUseItemType.HiEther), UpperRoyalTower & DoubleJumpOfNpc); //better describes "awkward not-really-a tower", also funnier
```

Comment:
I don't get what a Weenie hut is and I would have no idea where this is on the map.